ordered_product/views.py

```diff
@@ -0,0 +1,51 @@
+from django.shortcuts import render
+from .models import OrderedProduct
+from client.models import Client
+from supplier.models import Supplier
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from django.shortcuts import redirect
+
+
+def order_list(request):
+    if isinstance(request.user, Supplier):
+        return redirect('Main Page')
+    ordered_products = OrderedProduct.filter_user_name(Client(request.user))
+    return render(request, 'ordered_product/orders.html', {'orders': ordered_products})
+
+
+def change_quantity(request, order_id):
+    quantity = request.GET['quantity']
```

Comment:
```suggestion
    quantity = int(request.GET['quantity'])
```
Will reduce clutter from all uses of `quantity` and will also allow you to fail in one place if the call to `int()` fails.